plugins/rbac-common/src/types.ts

```diff
@@ -4,10 +4,15 @@ export type Policy = {
   effect?: string;
 };
 
-export type EntityReferencedPolicy = Policy & {
+export type RoleBasedPolicy = Policy & {
   entityReference?: string;
 };
 
+export type Role = {
+  entityReferences: string[];
```

Comment:
I think we need to provide more clear name to explain, that here is stored references to role users and groups.

Reply:
What do you think would be best? Maybe something like userReferences or groupReferences?

Reply:
Maybe roleMemberReferences, since it can be users and group in the same time.